examples_tests/__init__.py

```diff
@@ -178,9 +181,7 @@ def run_command_in_snappy_testbed(self, command):
     def assert_service_running(self, snap, service):
         if not config.get('skip-install', False):
             output = self.run_command_in_snappy_testbed(
-                ['sudo', 'snap', 'service', 'status', snap])
-            expected = (
-                'Snap\t+Service\t+State\n'
-                '{}\t+{}\t+enabled; loaded; active \(running\)\n'.format(
-                    snap, service))
-            self.assertThat(output, MatchesRegex(expected))
+                ['systemctl', '--no-pager', 'status',
+                 'snap.{}.{}'.format(snap, service)])
```

Comment:
This is the unit name, and yes, it changed. I don't remember how was it named before; but that shouldn't matter, all of these details should be hidden behind a nice snap command. snap service was one of the commands I liked the most, I hope they bring it back.